### PR TITLE
[DataZoom] new integration

### PR DESCRIPTION
**Pull request by ls339**
### What does this PR do?

New integration tile and OOTB dashboard for `datazoom` 

### Motivation

What inspired you to submit this pull request?

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

**Review by BoyangHuang — APPROVED** [review]

File: datazoom/manifest.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Datadog-Datazoom Integration",
+  "categories": [
+    ""
+  ],
+  "type": "check",
+  "is_public": false,
```
Comment (ls339) [inline datazoom/manifest.json:22]: Not yet GA.

File: datazoom/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Agent Check: datazoom
```
Comment (cswatt) [inline datazoom/README.md:1]:
```suggestion
# Agent Check: Datazoom
```

File: datazoom/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Overview
+
+The [Datazoom Datadog Connector][1] sends Collector data to a configured Datadog destination, available for query in Log Explorer.
```
Comment (cswatt) [inline datazoom/README.md:5]:
```suggestion
Datazoom is a video data platform that gathers data from endpoints through an ecosystem of collectors.

The [Datazoom Datadog Connector][1] sends collector data to Datadog, where you can query the data in your [Log Explorer][3].

Datazoom sends data set at the INFO level.
```

File: datazoom/README.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+### Events
+
+Datazoom does not include any events.
+
+## Troubleshooting
+
+Need help? Contact [Datadog support][2].
+
+[1]: https://help.datazoom.io/hc/en-us/articles/360042494512-Datadog
+[2]: https://docs.datadoghq.com/help/
```
Comment (cswatt) [inline datazoom/README.md:40]:
```suggestion
[2]: https://docs.datadoghq.com/help/
[3]: https://app.datadoghq.com/logs
[4]: https://app.datadoghq.com/dashboard/lists/preset/3?q=datazoom
```

File: datazoom/README.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+The Datazoom integration emits logs to Datadog.
+
+### Configuration
+
+- Visit Datazoom's integation [documentation][1].
+
+### Dashboard
+
+Datazoom logs dashboard is available [here](../dashboard/lists/preset/3?q=datazoom).
```
Comment (cswatt) [inline datazoom/README.md:19]:
```suggestion
See the [Datazoom logs dashboard][4].
```

File: datazoom/README.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+## Overview
+
+The [Datazoom Datadog Connector][1] sends Collector data to a configured Datadog destination, available for query in Log Explorer.
+
+## Setup
+
+### Installation
+
+The Datazoom integration emits logs to Datadog.
```
Comment (cswatt) [inline datazoom/README.md:11]:
```suggestion
The Datazoom integration emits logs to Datadog. No installation is required on the Datadog side.
```

File: datazoom/README.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+### Configuration
+
+- Visit Datazoom's integation [documentation][1].
```
Comment (cswatt) [inline datazoom/README.md:15]:
```suggestion
- Visit Datazoom's integation [documentation][1] for details on how to configure the Datazoom Datadog Connector.
```

**Review by cswatt — CHANGES_REQUESTED** [review]
left some comments to clean up the readme

**Review by cswatt — APPROVED** [review]

**Pull request merged**